### ISSUE TITLE
Support for retrieving forecast data

### ISSUE THREAD

**Issue by gerritsj** [DigitaleDeltaOrg/dd-api#37]
It should be possible to request the available analyses of a specific source. Moreover it should be possible to request the time series of a specific analysis.

**Comment by gerritsj** [comment]
HydroLogic (Sander/Bram) is interested in brainstorming about the design.

**Comment by GeriWolters** [comment]
Is er voortgang?

**Comment by gerritsj** [comment]
Nee, daarvoor moet eerst duidelijkheid komen over financiering (uit het DD overleg van vandaag).
